how-to/misc/generate-a-ca-certificate.rst

```diff
@@ -0,0 +1,161 @@
+Generating a CA Certificate for TLS Vault
```

Comment:
Can you move this page to https://github.com/canonical/snap-openstack/tree/main/sunbeam-python/sunbeam/features/tls since this procedure is for testing purpose and not intended for Users.